include/odp/api/spec/packet.h

```diff
@@ -2086,21 +2086,21 @@ int odp_packet_has_tx_compl_request(odp_packet_t pkt);
 /**
  * Set packet free control option
  *
- * This option enables application to control which packets are freed/not freed back into pool
- * after ODP implementation has finished processing those. The option affects only packets that
- * are transmitted directly through a packet output interface (also with LSO), i.e. packets
- * transmitted through inline IPsec or TM are not affected.
- *
- * When the option is set to #ODP_PACKET_FREE_CTRL_DONT_FREE, packet output interface will not free
- * the packet after transmit and application may reuse the packet as soon as its transmit is
- * complete (see e.g. odp_packet_tx_compl_done()).
- *
- * The option must not be enabled on packets that have multiple references.
- *
- * Check packet IO interface capability free_ctrl.dont_free (odp_pktio_capability_t::dont_free) for
- * the option support. When an interface does not support the option, it ignores the value.
- *
- * The default value is #ODP_PACKET_FREE_CTRL_DISABLED.
+ * This option enables application to control if a packet is freed/not freed back into pool
+ * after an ODP offload feature has finished processing it. The option does not have effect on
+ * odp_packet_free() or other direct free calls.
+ *
+ * Packet output interface frees transmitted packets by default. This option affects only packets
+ * that are sent directly (see e.g. odp_pktout_send()) through an interface (also with LSO).
```

Comment:
Should this be "... sent *or enqueued* directly ...", for further clarity if enqueued to a packet I/O event queue?

Reply:
Added queue enqueue option there with a bit of rewrite in v2. Please check.